taxi/forms.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "license_number", "first_name", "last_name", "email"
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta(UserCreationForm.Meta):

```

Comment:
The `Meta` class in `DriverLicenseUpdateForm` should inherit from `forms.ModelForm.Meta` instead of `UserCreationForm.Meta`. This is because `DriverLicenseUpdateForm` is a `ModelForm`, not a `UserCreationForm`.